x/campaign/keeper/invariants.go

```diff
@@ -140,14 +132,9 @@ func CampaignSharesInvariant(k Keeper) sdk.Invariant {
 				vouchers = vouchers.Add(voucherSupply)
 			}
 
-			// convert to shares and add to the campaign shares
-			vShares, err := types.VouchersToShares(vouchers, campaignID)
-			if err != nil {
-				return sdk.FormatInvariant(
-					types.ModuleName, campaignSharesRoute,
-					"fail to convert vouchers to shares",
-				), true
-			}
+			// convert to shares and add to the campaign shares - since we are converting shares to vouchers earlier,
+			// this conversion back to shares will never fail by design, thus we can ignore the error
+			vShares, _ := types.VouchersToShares(vouchers, campaignID)
```

Comment:
As commented, we can just avoid checking the error. This conversion cannot fail (see L117 and the use of `types.SharesToVouchers`). I can re-add it but it would be dead code

Reply:
You can add a `//nolint` message here, perhaps